weldr/compose.go

```diff
@@ -348,14 +348,20 @@ func (c Client) ComposeMetadata(id string) (fileName string, apiResponse *APIRes
 	return c.GetFilePath(route, "")
 }
 
-// ComposeResults returns the tar file of compose's results
-// It returns a temporary filename, the name of the file from the server, and the type.
-// The caller must clean up the temporary file when finished
+// ComposeResults saves the compose's results to a file in the current directory
+// It returns the filename, the server response, and the error.
 func (c Client) ComposeResults(id string) (fileName string, apiResponse *APIResponse, err error) {
 	route := fmt.Sprintf("/compose/results/%s", id)
 	return c.GetFilePath(route, "")
 }
 
+// ComposeResultsPath saves the compose's results to a directory or file in path
+// It returns the filename, the server response, and the error.
+func (c Client) ComposeResultsPath(id, path string) (fileName string, apiResponse *APIResponse, err error) {
+	route := fmt.Sprintf("/compose/results/%s", id)
+	return c.GetFilePath(route, path)
+}
+
```

Comment:
I suppose these two functions could be one (it seems it's only called from one place?) or at least the older one could be a wrapper around the new one (even if it's just 2 lines).

Reply:
Good point. I'm not sure why I didn't make it a wrapper :)